rules/scalafmt/private/test.bzl

```diff
@@ -71,31 +73,37 @@ def build_format(ctx):
     return manifest, files
 
 def format_runner(ctx, manifest, files):
+    args = ctx.actions.args()
+    args.add(ctx.file._runner)
+    args.add(ctx.workspace_name)
+    args.add(manifest.short_path)
```

Comment:
Shouldn't this be `args.add([manifest], map_each = _short_path)`.

Reply:
Same comment response as above.